fava/static/javascript/helpers.js

```diff
@@ -81,3 +83,11 @@ export function _(string, vars = {}) {
   const translation = window.favaTranslations[string] || string;
   return translation.replace(/\${([a-z]+)}/gi, (match, v) => vars[v] || v);
 }
+
+// Format a currency
+export function formatCurrency(num, fstr = '.2f') {
+  if (window.favaAPI.incognito) {
+    return format(fstr)(num).replace(/[0-9]/g, 'X');
+  }
+  return format(fstr)(num);
```

Comment:
Ahh thanks for the hint - I could just store them in variables, and send them out by the `formatCurrency`-function.